### PR TITLE
MONIT-16650 - update Jaeger SDK

### PR DESCRIPTION
**Pull request by thepeterstone**
Update the Jaeger/OpenTracing SDK to v1.5.0 and update the instructions in the Readme.

### CODE REVIEW

File: distributed-tracing/dropwizard-app/README.md [hunk 1]
```diff
@@ -55,72 +55,66 @@ This is a sample Java application using Dropwizard called beachshirts (#[beachop
...
+         -e JAVA_HEAP_USAGE=512m \
+         -p 2878:2878 \
+         -p 4242:4242 \
+         wavefrontHQ/proxy:latest
      ```
 
 3. If you are sending data to Wavefront via Direct Ingestion, then make sure you have the cluster name and corresponding token from [https://{cluster}.wavefront.com/settings/profile](https://{cluster}.wavefront.com/settings/profile).
 
-4. Go to `dropwizard-app/common/../Tracing.java` and change the `Tracer init(String service)` method to return a [WavefrontTracer](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java#set-up-a-tracer) instead of `com.uber.jaeger.Tracer` as follows:
+4. Go to `dropwizard-app/common/src/main/java/com/wfsample/common/Tracing.java` and change the `Tracer init(String service)` method to return a [WavefrontTracer](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java#set-up-a-tracer) instead of `io.opentracing.Tracer` as follows:
```
Comment (hanwavefront) [inline distributed-tracing/dropwizard-app/README.md:76]: Replace `io.opentracing.Tracer` with `JaegerTracer`

File: distributed-tracing/dropwizard-app/common/src/main/java/com/wfsample/common/Tracing.java [hunk 2]
```diff
@@ -31,7 +26,7 @@
   private Tracing() {
   }
 
-  public static com.uber.jaeger.Tracer init(String service) {
+  public static io.opentracing.Tracer init(String service) {
```
Comment (hanwavefront) [inline distributed-tracing/dropwizard-app/common/src/main/java/com/wfsample/common/Tracing.java:29]: `Tracer` doesn't need to be fully qualified name here

File: distributed-tracing/dropwizard-app/pom.xml [hunk 3]
```diff
@@ -23,15 +23,25 @@
...
+       <dependency>
+            <groupId>io.jaegertracing</groupId>
+            <artifactId>jaeger-thrift</artifactId>
+            <version>${jaeger.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>4.9.0</version>
+        </dependency>
```
Comment (hanwavefront) [inline distributed-tracing/dropwizard-app/pom.xml:44]: Is okhttp still needed?

**Review by hanwavefront — CHANGES_REQUESTED** [review]
A couple minor comments, otherwise LGTM, thanks!

**Review by hanwavefront — APPROVED** [review]
LGTM

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**